lib/src/resources/movie_api_provider.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+import 'package:http/http.dart' show Client;
+import 'package:movies_bloc/src/styles/string_cons.dart';
```

Comment:
You missed to changed to relative import here

Reply:
done